.claude/commands/01-project-setup/bootstrap-project.md

```
@@ -0,0 +1,57 @@
+---
+description: Complete project initialization with intelligent technology detection
+allowed-tools: Bash(npm:*), Bash(pip:*), Bash(cargo:*), Bash(git:*), Write, Edit, Read
+---
+
+# Bootstrap Project
+
+## Project Analysis
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)